tests/test_serialization.py

```diff
@@ -494,11 +516,11 @@ def test_augmentations_for_bboxes_serialization(
     serialized_aug = A.to_dict(aug)
     deserialized_aug = A.from_dict(serialized_aug)
     set_seed(seed)
-    aug_data = aug(image=image, bboxes=albumentations_bboxes)
+    aug_data = aug(image=image, bboxes=to_internal_bboxes(albumentations_bboxes))
```

Comment:
because bboxes are not converted to internal type unless they were passed through A.Compose.
BasicTransform does not transform data to internal type, only Compose does.